### PR TITLE
Story/74953 - HOM 2

### PR DESCRIPTION
**Pull request by thiiagofernando**
[AB#76637](https://dev.azure.com/amcomgov/c1dcf343-60ee-40b6-a30a-3b9c12c6d220/_workitems/edit/76637)

### CODE REVIEW

File: SME.Pedagogico.Gestao.Data/Relatorios/Querys/ConsultasRelatorios.cs [hunk 1]
```diff
@@ -8,7 +8,10 @@ namespace SME.Pedagogico.Gestao.Data.Relatorios.Querys
     public static class ConsultasRelatorios
     {
 
-        private const int TERCEIRO_ANO = 3;
+	    private const int TERCEIRO_ANO = 3;
+	    private const int QUARTO_ANO = 4;
```
Comment (viniciusnyari) [inline SME.Pedagogico.Gestao.Data/Relatorios/Querys/ConsultasRelatorios.cs:12]: O mesmo caso das demais

**Review by Jailsvs — APPROVED** [review]

File: SME.Pedagogico.Gestao.Data/Business/SondagemAutoralBusiness.cs [hunk 2]
```diff
@@ -21,6 +21,9 @@ public class SondagemAutoralBusiness
     {
         private readonly IServicoTelemetria servicoTelemetria;
         private readonly string connectionString;
+        private const int QUARTO_ANO = 4;
+        private const int NONO_ANO = 9;
+        private const int QUARTO_BIMESTRE = 4;
```
[the remark this answers is not in the document]
Comment (thiiagofernando) [reply SME.Pedagogico.Gestao.Data/Business/SondagemAutoralBusiness.cs:26]: Alterado

File: SME.Pedagogico.Gestao.Data/Business/SondagemMatematica.cs [hunk 3]
```diff
@@ -24,6 +24,9 @@ public class SondagemMatematica
         private AlunosAPI alunoAPI;
         public IConfiguration _config;
         private TurmasAPI TurmaApi;
+        private const int QUARTO_ANO = 4;
+        private const int NONO_ANO = 9;
```
[the remark this answers is not in the document]
Comment (thiiagofernando) [reply SME.Pedagogico.Gestao.Data/Business/SondagemMatematica.cs:28]: Alterado

File: SME.Pedagogico.Gestao.Data/Relatorios/Querys/ConsultasRelatorios.cs (hunk 1 above)
Comment (thiiagofernando) [reply SME.Pedagogico.Gestao.Data/Relatorios/Querys/ConsultasRelatorios.cs:12]: Alterado

**Review by viniciusnyari — APPROVED** [review]

**Pull request merged**